ldap/servers/plugins/syntaxes/syntax_common.c

```diff
@@ -16,7 +16,7 @@ int
 syntax_register_matching_rule_plugins(
     struct mr_plugin_def mr_plugin_table[],
     size_t mr_plugin_table_size,
-    IFP matching_rule_plugin_init)
+    int32 (*matching_rule_plugin_init)(Slapi_PBlock *))
```

Comment:
`int32_t` typo

Also, `matching_rule_plugin_init` is defined as `int` everywhere, but I don't have a hard opinion on its consistency.

Reply:
> `int32_t` typo
> 
> Also, `matching_rule_plugin_init` is defined as `int` everywhere, but I don't have a hard opinion on its consistency.

Yeah trying to not to make this patch bigger than it needs to be.  To be consistent I made all function pointer declarations use int32_t, but to update all the calling functions would be a lot of work.